Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -320,11 +332,8 @@ def get_full_report(
         :param id: id of a submitted malware samples.
         :return: http response
         """
-        url_suffix = f"/falconx/entities/reports/v1?ids={id}"
-        params = {
-            "ids": id
-        }
-        return self._http_request("Get", url_suffix, params=params)
+        url_suffix = "/falconx/entities/reports/v1"
+        return self._http_request("Get", url_suffix, params={"ids": id})
```

Comment:
Since we're in the spirit of refactoring, please consider making this a one-liner, and of course all the rest of the HTTP class methods. Your call.